### PR TITLE
Added health check (redash-server) and updated README.md

### LINKED ISSUE #36: Health checks in Consul
## Feature description
Add health checks:
- Redash server
- Redash worker
- Redash scheduler

## Checklist (after created issue)
- [x] Added label(s)
- [ ] Added to project
- [x] Added to milestone

### LINKED ISSUE #38: Update documentation for /example/redash_one_node
## Feature description
Update [README.md](https://github.com/Skatteetaten/terraform-nomad-redash/blob/master/example/redash_one_node/README.md)

## Checklist (after created issue)
- [ ] Added label(s)
- [ ] Added to project
- [ ] Added to milestone

### LINKED ISSUE #40: Update documentation for /example/redash_trino_cluster
## Feature description
Update README.md

## Checklist (after created issue)
- [ ] Added label(s)
- [ ] Added to project
- [ ] Added to milestone

### PR DESCRIPTION
**Pull request by hannemariavister**
## Closes/fixes/resolves issue(s)?
Closes #36 
Closes #38 
Closes #40 

## What was added/changed/fixed?
- Documentation for example/redash_trino_cluster #40
- Documentation for example/redash_one_node #38
- Added health checks for redash-server #36

## Checklist (after created PR)
- [x] Added reviewers
- [ ] Added assignee(s)
- [x] Added label(s)
- [ ] Added to project
- [x] Added to milestone
- Need to update documentation? (Y/n)

### CODE REVIEW

**Review by AlexanderSkjolden65536 — APPROVED** [review]
LGTM 👍

File: README.md [hunk 1]
```diff
@@ -97,14 +112,12 @@ module "postgres" {
...
   container_port  = 5432
   vault_secret = {
-    use_vault_provider      = false,
-    vault_kv_policy_name    = "",
-    vault_kv_path           = "",
-    vault_kv_field_username = "",
-    vault_kv_field_password = ""
+    use_vault_provider      = false
+    vault_kv_policy_name    = "kv-secret"
+    vault_kv_path           = "secret/data/dev/postgres"
```
Comment (skatteknut) [inline README.md]: Noen grunn til at dette avviker fra innholdet på linje 83?

**Review by skatteknut — APPROVED** [review]
:+1:

File: README.md (hunk 1 above)
Comment (hannemariavister) [reply README.md]: Bra sett! Rettet nå 😉

**Pull request merged**